### PR TITLE
FAQ.md

### PR DESCRIPTION
**Pull request by codiwest**

### CODE REVIEW

File: docs/FAQ.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+### Contents
+
+* [Will I make money with Zenbot?](#money)
+* [Is Docker necessary when installing Zenbot?](#docker)
+* [Why do simulations, paper trading, and live trading all yield different results?](#sim-live-differences)
+* [Why should I use simulations or paper trading if they do not reflect live trading?](#why-sim)
+* [Does Zenbot use Limit orders or Market orders?](#limit-market-orders)
+* [Why does Zenbot use Limit orders by default?](#limit-orders)
+* [Is there a community to get involved with Zenbot?](#community)
+* [Why do I keep getting a this "Nonce" error?](#error-1)
```
Comment (DeviaVir) [inline docs/FAQ.md]: I think this one is also often popping up:
`* [How do I update zenbot?]`

docker: `git pull && docker-compose down && docker rmi zenbot_server && docker-compose build && docker-compose up -d`
standalone: `git pull && npm install && ./zenbot.sh trade --paper`

File: docs/FAQ.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+## Frequently Asked Questions
+
+Here are a few questions that have been asked often as well as answers from the community.
+
+If you have a question that is not answered here, feel free to ask in the Discord chat and suggest it to be placed here.
```
Comment (DeviaVir) [inline docs/FAQ.md]: maybe link `discord` here to `#community`?

File: docs/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+* [Why should I use simulations or paper trading if they do not reflect live trading?](#why-sim)
+* [Does Zenbot use Limit orders or Market orders?](#limit-market-orders)
+* [Why does Zenbot use Limit orders by default?](#limit-orders)
+* [Is there a community to get involved with Zenbot?](#community)
+* [Why do I keep getting a this "Nonce" error?](#error-1)
+
+
+**Will I make money with Zenbot?** <a name="money"></a>
+
+That depends… a lot. Different configurations and strategies will yield different results.
```
Comment (DeviaVir) [inline docs/FAQ.md]: `a lot` seems a bit out of place here, "That depends..." or "That depends on a lot..." sort of make sense, this doesn't?

File: docs/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+**Does Zenbot use Limit orders or Market orders?** <a name="limit-market-orders"></a>
+
+By default, Zenbot will use Limit orders, but support for Market orders has been added. 
+
+
+
+**Why does Zenbot use Limit orders by default?** <a name="limit-orders"></a>
+
+On most exchanges, Limit orders result in lower fees than Market orders. For instance, on GDAX there no fee for a Limit order trade compared to a 0.25% (BTC) or 0.3% (ETH & LTC) trade fee on a Market order.
```
Comment (DeviaVir) [inline docs/FAQ.md]: `there no fee` => `there's no fee`

File: docs/FAQ.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+Simulations are more optimistic than paper trading.
+Paper trading is more optimistic than live trading.
+Therefore, if a simulation does not yield good results, odds are that neither will paper trading or (by extension) live trading.
+
+
+
+**Does Zenbot use Limit orders or Market orders?** <a name="limit-market-orders"></a>
+
+By default, Zenbot will use Limit orders, but support for Market orders has been added.
```
Comment (DeviaVir) [inline docs/FAQ.md]: Not sure if FAQ-worthy: perhaps give a bit more info about the difference between limit order (maker) and market order (taker)?

File: docs/FAQ.md [hunk 6]
```diff
@@ -0,0 +1,83 @@
...
+[![zenbot logo](https://rawgit.com/carlos8f/zenbot/master/assets/discord.png)](https://discord.gg/ZdAd2gP)
+
+There is also a shared Google Doc spreadsheet containing community sim results and variable descriptions:
+[Click here for the Google Sheet](https://docs.google.com/spreadsheets/d/1WjFKRUY4KpkdIJiA3RVvKqiyNkMe9xtgLSfYESFXk1g/edit#gid=70204991).
+
+### Errors
+
+**Why do I keep getting this "Nonce" error? How do I solve it?** <a name="error-1"></a>
+
+This error may occur when multiple instances of Zenbot are used with the same API key. To avoid this, use a different API key for each concurrent instance of Zenbot.
```
Comment (DeviaVir) [inline docs/FAQ.md:123]: It is also possible that the server's time is not current, you could install NTP to solve this.

**Review by DeviaVir — COMMENTED** [review]
Very happy to see this being made! Definitely something that will need to be added to the `README.md` as a link as well, as stickied to the Discord #general topic.

**Comment by DeviaVir** [comment]
via https://github.com/carlos8f/zenbot/issues/253

Another FAQ is Windows users:

# Windows
1. Install the "Bash on Windows subsystem" (see https://msdn.microsoft.com/en-us/commandline/wsl/about for more information)
> Note: This will change soon, seeing that Microsoft decided to have Ubuntu, Fedora and OpenSUSE available. They will also drop the requirement for requiring dev mode enabled on windows soon. Source: https://blogs.msdn.microsoft.com/commandline/2017/05/11/new-distros-coming-to-bashwsl-via-windows-store/ and https://blogs.msdn.microsoft.com/commandline/2017/06/08/developer-mode-no-longer-required-for-windows-subsystem-for-linux/
2. Open the terminal window, your disks will now be mounted under `/mnt/*`. So for example navigate to your directory (example: `cd /mnt/c/zenbot`)
3. Install node (`curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -; sudo apt-get install nodejs)
4. Run zenbot as you would otherwise

File: docs/FAQ.md (hunk 1 above)
Comment (codiwest) [reply docs/FAQ.md]: I'm curious; what is the `./zenbot.sh trade --paper` used for? To test that it is working?

Comment (DeviaVir) [reply docs/FAQ.md]: Yep, the paper mode uses live data to simulate trades. Sim uses history data.

**Review by DeviaVir — APPROVED** [review]
:+1:

**Pull request merged**